miniserver/blockchain.js

```diff
@@ -0,0 +1,124 @@
+const config = require('./config')
+const { ethers } = require('ethers')
+const { Logger } = require('./logger')
+const cloneDeep = require('lodash/fp/cloneDeep')
+const { backOff } = require('exponential-backoff')
+const { rpc } = require('./rpc')
+const MiniWallet = require('../miniwallet/build/contracts/MiniWallet.sol/MiniWallet.json')
+
+let networkConfig = {}
+let provider
+const pendingNonces = {}
+const signers = []
+const miniWallets = []
+const walletPath = 'm/44\'/60\'/0\'/0/' // https://docs.ethers.io/v5/api/signer/#Wallet.fromMnemonic'
```

Comment:
use backquotes `` to avoid confusion of string escape

Reply:
also can move to constants

Reply:
done